src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/__tests__/ExposureSubmissionCoordinatorModelTests.swift

```diff
@@ -178,49 +202,48 @@ class ExposureSubmissionCoordinatorModelTests: XCTestCase {
 		}
 	}
 
-	func testSymptomsOptionNoOrPreferNotToSaySelectedSupportedCountriesLoadFails() {
-		let symptomOptions: [ExposureSubmissionSymptomsViewController.SymptomsOption] = [.no, .preferNotToSay]
-		for symptomOption in symptomOptions {
-			let exposureSubmissionService = MockExposureSubmissionService()
-			exposureSubmissionService.submitExposureCallback = { completion in
-				completion(nil)
-			}
-
-			let mockAppConfiguration = MockAppConfiguration(mockSAPApplicationConfiguration: nil)
-
-			let model = ExposureSubmissionCoordinatorModel(
-				exposureSubmissionService: exposureSubmissionService,
-				appConfigurationProvider: mockAppConfiguration
-			)
-
-			let expectedIsLoadingValues = [true, false]
-			var isLoadingValues = [Bool]()
-
-			let isLoadingExpectation = expectation(description: "isLoading is called twice")
-			isLoadingExpectation.expectedFulfillmentCount = 2
-
-			let onSuccessExpectation = expectation(description: "onSuccess is not called")
-			onSuccessExpectation.isInverted = true
-
-			let onErrorExpectation = expectation(description: "onError is called")
-
-			model.symptomsOptionSelected(
-				selectedSymptomsOption: symptomOption,
-				isLoading: {
-					isLoadingValues.append($0)
-					isLoadingExpectation.fulfill()
-				},
-				onSuccess: { onSuccessExpectation.fulfill() },
-				onError: { _ in onErrorExpectation.fulfill() }
-			)
-
-			waitForExpectations(timeout: .short)
-			XCTAssertEqual(isLoadingValues, expectedIsLoadingValues)
-
-			XCTAssertFalse(model.shouldShowSymptomsOnsetScreen)
-			XCTAssertEqual(model.supportedCountries, [])
-		}
-	}
+//	func testSymptomsOptionNoOrPreferNotToSaySelectedSupportedCountriesLoadFails() {
+//		let symptomOptions: [ExposureSubmissionSymptomsViewController.SymptomsOption] = [.no, .preferNotToSay]
```

Comment:
Hmm is this still needed or can it be removed?

Reply:
I forgot to check with @nickguendling on that! I don't understand the logic that denying this should affect the local cache.

Reply:
fixed & reenabled